plugins/main/windows/l2bridge/sample.conf

```diff
@@ -0,0 +1,44 @@
+{
+    "name":"cbr0",
+    "type":"flannel",
+    "delegate":{
+        "type":"win-l2bridge",
+        "dns":{
+            "nameservers":[
+                "11.0.0.10"
+            ],
+            "search":[
+                "svc.cluster.local"
+            ]
+        },
+        "additionalArgs":[

```

Comment:
This should be a proper field in the configuration type, not AdditionalArgs. Even better would be a field for each kind of policy (e.g. nat, route). If that's not reasonable, there should be a specific `policy:` field.